packages/grid/x-data-grid/src/locales/itIT.ts

```diff
@@ -71,33 +71,33 @@ const itITGrid: Partial<GridLocaleText> = {
   filterOperatorIsEmpty: 'è vuoto',
   filterOperatorIsNotEmpty: 'non è vuoto',
   filterOperatorIsAnyOf: 'è uno tra',
-  // 'filterOperator=': '=',
-  // 'filterOperator!=': '!=',
-  // 'filterOperator>': '>',
-  // 'filterOperator>=': '>=',
-  // 'filterOperator<': '<',
-  // 'filterOperator<=': '<=',
+  'filterOperator=': '=',
+  'filterOperator!=': '!=',
+  'filterOperator>': '>',
+  'filterOperator>=': '>=',
+  'filterOperator<': '<',
+  'filterOperator<=': '<=',
 
   // Header filter operators text
-  // headerFilterOperatorContains: 'Contains',
-  // headerFilterOperatorEquals: 'Equals',
-  // headerFilterOperatorStartsWith: 'Starts with',
-  // headerFilterOperatorEndsWith: 'Ends with',
-  // headerFilterOperatorIs: 'Is',
-  // headerFilterOperatorNot: 'Is not',
-  // headerFilterOperatorAfter: 'Is after',
-  // headerFilterOperatorOnOrAfter: 'Is on or after',
-  // headerFilterOperatorBefore: 'Is before',
-  // headerFilterOperatorOnOrBefore: 'Is on or before',
-  // headerFilterOperatorIsEmpty: 'Is empty',
-  // headerFilterOperatorIsNotEmpty: 'Is not empty',
-  // headerFilterOperatorIsAnyOf: 'Is any of',
-  // 'headerFilterOperator=': 'Equals',
-  // 'headerFilterOperator!=': 'Not equals',
-  // 'headerFilterOperator>': 'Greater than',
-  // 'headerFilterOperator>=': 'Greater than or equal to',
-  // 'headerFilterOperator<': 'Less than',
-  // 'headerFilterOperator<=': 'Less than or equal to',
+  headerFilterOperatorContains: 'Contiene',
+  headerFilterOperatorEquals: 'uguale a',
+  headerFilterOperatorStartsWith: 'comincia per',
+  headerFilterOperatorEndsWith: 'termina per',
+  headerFilterOperatorIs: 'uguale a',
+  headerFilterOperatorNot: 'diversa da',
+  headerFilterOperatorAfter: 'dopo il',
+  headerFilterOperatorOnOrAfter: 'a partire dal',
+  headerFilterOperatorBefore: 'prima del',
+  headerFilterOperatorOnOrBefore: 'fino al',
+  headerFilterOperatorIsEmpty: 'è vuoto',
+  headerFilterOperatorIsNotEmpty: 'non è vuoto',
+  headerFilterOperatorIsAnyOf: 'è uno tra',
+  'headerFilterOperator=': '=',
```

Comment:
Can we have the textual values here and the following ones? It will make more sense as the values are meant to be shown in the dropdown select for [header filters](https://mui.com/x/react-data-grid/filtering/header-filters/).

Reply:
Sure